doc/Language/intro.pod6

```diff
@@ -0,0 +1,46 @@
+=begin pod
+
+=TITLE Brief Introduction
+
+=SUBTITLE Using the Perl6 Documentation
```

Comment:
This whole document says “Perl6” when I think we settled on “Perl 6” with no-breaking space. Maybe it is worth changing it, although with recent discussions regarding an alias it may be worth to reword some sentences so that the language name is not spelled out. @JJ what do you think?